test/helpers/module-setup.ts

```diff
@@ -39,7 +39,7 @@ export async function resetTestModule(testModulePath: string, installModules = t
 }
 
 export async function cleanupTestModule(testModulePath: string): Promise<void> {
-  await fs.rmdir(testModulePath, { recursive: true, maxRetries: 10 });
+  await fs.rm(testModulePath, { recursive: true, maxRetries: 10 });
```

Comment:
Clears a warning that was showing up during test runs about `fs.rmdir({ recursive: true })` being deprecated.